plasmapy/formulary/distribution.py

```diff
@@ -104,27 +100,22 @@ def Maxwellian_1D(v, T, particle="e", v_drift=0, vTh=np.nan, units="units"):
     >>> Maxwellian_1D(v=v, T=30000 * u.K, particle='e', v_drift=0 * u.m / u.s)
     <Quantity 5.9163...e-07 s / m>
     """
+    # unit checks and conversions
+    # checking velocity units
+    # v = v * (u.m / u.s)
+    v = v.to(u.m / u.s)
+    # Catching case where drift velocities have default values, they
+    # need to be assigned units
+    v_drift = v_drift.to(u.m / u.s)
+    # convert temperature to Kelvins
+    # T = T.to(u.K, equivalencies=u.temperature_energy())
```

Comment:
The `validate_quantities` decorator you're using above should mostly handle this block, I think :)